db/migrate/20230502104738_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
```

Comment:
- [ ] Let's make sure that the data in our migration files reflect what we have on the ERD. In this case, we want to name our user foreign keys as __author_id__. By default, this can bring issues because Rails only knows a user (user_id). We need to explicitly tell Rails that a user will be called __author__ and its __id__ will now be __author_id__, __HINT__:
```ruby
    t.references :author, foreign_key: { to_table: 'users' } # This will create author_id and a corresponding index 
```
[Read this documentation for reference](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)

Reply:
@ram1117, I got this ERD from this file which happens to be the requirements file for all students [Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md)